lib/galaxy/tools/parameters/basic.py

```diff
@@ -2216,20 +2216,20 @@ def value_to_display_text( self, value, app ):
     def validate( self, value, trans=None ):
         return True  # TODO
 
-    def to_dict( self, trans, view='collection', value_mapper=None, other_values=None ):
+    def to_dict( self, trans, view='collection', value_mapper=None, other_values={} ):
```

Comment:
Not quite sure about this change to the method signature.  Wouldn't it be better to use the original where other_values=None (which is immutable), then in the method check if other_values is None and set to an empty dict?